appliance-root/opt/napp/bin/broker-start

```diff
@@ -81,5 +81,8 @@ if [ -n "`grep '^    <circonus>' $CONF`" ]; then
 	sed -ie 's#^    <circonus>#    <circonus backingstore="/opt/noit/prod/etc/filtersets">#;' $CONF
 fi
 
+test ! -f /opt/napp/etc/ssl/graphite.key
```

Comment:
I would probably check for the .crt instead, since it could happen that we are still waiting for a cert to be issued/installed.